docs/examples.rst

```diff
@@ -101,7 +101,7 @@ Intraday examples
   specific contract. (`[view on github] <https://github.com/volueinsight/volue-insight-timeseries/blob/development/examples/intraday_examples/absolute_intraday_price_forecast.py>`_ ,
   :download:`[download] <../examples/intraday_examples/absolute_intraday_price_forecast.py>`)
 * Example on how to create lead time analysis plots. These plots show how the difference between a chosen benchmark
-forecast and one of our intraday forecasts develops over time (`[view on github] <https://github.com/volueinsight/volue-insight-timeseries/blob/development/examples/intraday_examples/absolute_intraday_price_forecast.py>`_ ,
+  forecast and one of our intraday forecasts develops over time (`[view on github] <https://github.com/volueinsight/volue-insight-timeseries/blob/development/examples/intraday_examples/absolute_intraday_price_forecast.py>`_ ,
```

Comment:
This is a link to the "absolute_intraday_price_forecast.py" script. You should link the lead time analysis instead.

Also, the mergeable check is failing, so you need to rename the PR.